src/platyPS/platyPS.psd1

```diff
@@ -18,19 +18,19 @@ ModuleVersion = '0.0.1'
 GUID = '0bdcabef-a4b7-4a6d-bf7e-d879817ebbff'
 
 # Author of this module
-Author = 'PowerShell team'
+Author = 'Microsoft Corporation'
```

Comment:
what's the reason to change that?

Reply:
In PowerShell repo, we have Author = `PowerShell`, we can keep that consistent here.

Reply:
I'll update the author to `PowerShell`.